SU-SWS/stanford_profile
# D8CORE-931: Add anchor location for skip links to content layouts

# READY FOR REVIEW

# Summary
- Add anchor block from skip links.
- Removes ds_1col layout as it is an invalid layout for the allowed list.

# Needed By (Date)
- End of next week. (Feb 6)

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Do a full fresh install
3. Validate that all pages have the skip to main content anchor block in their layout.

# Affected Projects or Products
- Accessibility
- D8CORE-931

# Associated Issues and/or People
- D8CORE-931

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
